src/rules/no-snapshots.js

```diff
@@ -0,0 +1,37 @@
+import { getDocsUrl } from './util';
+
+export default {
+  meta: {
+    docs: {
+      description: 'Disallow snapshots',
+      uri: getDocsUrl(__filename),
+    },
+    messages: {
+      noSnapshots:
+        'Do not use {{method}} or related methods that generate snapshots.',
+    },
+  },
+
+  create(context) {
+    return {
+      CallExpression({ callee: { property } }) {
+        if (isSnapshot(property && property.name)) {
+          context.report({
+            messageId: 'noSnapshots',
+            node: property,
+            data: { method: property.name },
+          });
+        }
+      },
+    };
+  },
+};
+
+function isSnapshot(name) {
```

Comment:
can you keep this function outside of the export?

Reply:
@SimenB Might be missing something, but isn't it outside the export?